agent/eni/iphelperwrapper/iphelper_windows.go

```diff
@@ -0,0 +1,163 @@
+// +build windows
+
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package iphelperwrapper
+
+import (
+	"C"
+	"math"
+	"syscall"
+	"time"
+	"unsafe"
+
+	log "github.com/cihub/seelog"
+	"github.com/pkg/errors"
+	"golang.org/x/sys/windows"
+)
+
+const (
+	// After registering a callback with NotifyIPInterfaceChange API, we get an initial notification to confirm that the callback is registered
+	// All other notifications sent on the channel would be positive except this one
+	InitialNotification = math.MinInt16
+	// This is the timeout for the initial notification. If notification is not received within this timeout then we will assume error
+	InitialNotificationTimeout = 3 * time.Second
+	// This is the delay following the receipt of Initial Notification, after which the corresponding message is sent on the channel
+	// This has been done to ensure that the control reaches 'initialNotificationLoop' before the initial notification is sent
+	// Also, note that the initial notification will only be sent once. Therefore, this delay would be for the first communication only
+	DelayInCallbackMessageForInitialNotification = 10 * time.Millisecond
+)
+
+var (
+	moduleiphlpapi = windows.NewLazySystemDLL("iphlpapi.dll")
```

Comment:
Why are these scoped to the package? 
Can they not be part of the implementation?

Reply:
These have been scoped to package so as to inject their mocks during testing.
These are not exported and are lazily loaded.